docs/Tutorials/Advanced/ticketing/drop.md

```diff
@@ -0,0 +1,390 @@
+
+- `accountIdField`: Specifies what field Keypom should auto-inject the claiming account's `accountId`into when calling the function.  
+- `dropIdField`: Specifies what field Keypom should auto-inject the drop's `dropId` into when calling the function.  
+
+These parameters will be used alongisde the [`createNFTSeries`](../../../keypom-sdk/modules.md#createnftseries) SDK method.  
+
+In this case, the drop will be interfacing with an NFT contract deployed at `nft-v2.keypom.testnet`. This is a contract specifically made to work with the SDK to allow you to seamlessly create NFTs and NFT Series to attach to your drops. 
+
+:::note
+If you wish to use a different NFT contract for your POAP, ensure you know the contract's interface and tailor the `methods` arguments accordingly.
+:::
+
+```js reference
+https://github.com/keypom/keypom-js/blob/5e4b4744a16c727d96d235282020c186edd0b0b5/docs-advanced-tutorials/ticket-app/frontend/utils/createTickDrop.js#L30-L61
+```
+
+:::note
+Calling `claim` here before creating the NFT Series will fail. This only applies if you choose to include a POAP using `nft_mint` in the second `claim`. 
+:::
+
+ 
+
+You may have noticed that the syntax for the `nft_mint` function call looks slightly different from what was used in the [FC drop tutorial](../../Basics/fc-drops.md#creating-drop-with-function-call-data). This is because an NFT series is being minted. The NFT series must be defined before `claim` can be called and will require the drop ID, as you'll see in the next section. 
+
+:::info
+An NFT series can be thought of as a bucket of NFT token IDs that *all* share similar information. This information comes in the form of metadata, royalties, price etc. For more on NFT series, see NEAR's [NFT tutorial](https://docs.near.org/tutorials/nfts/series#nft-collections-and-series)
+:::
+
+
+## Making NFT Series
+In this section, you'll be creating the series of NFTs to be used as POAPs.
+
+The Keypom SDK provides a function to create an NFT series specifically for function call drops, called [`createNFTSeries`](../../../keypom-sdk/modules.md#createnftseries). It requires the following parameters: 
+
+- `dropId`: The drop ID for the drop that should have the NFT series associated with it.  
+- `metadata`: 	The metadata that all minted NFTs will have.  
+
+`metadata` is an object with these properties:
+* `title`: The title for the NFTs in the series.
+* `description`: Description for all NFTs in the series.
+* `media`: link to the media for the NFTs.
+* `copies`: Number of NFTs in the series.
+
+The code for creating the series is shown below. 
+
+```js reference
+https://github.com/keypom/keypom-js/blob/5e4b4744a16c727d96d235282020c186edd0b0b5/docs-advanced-tutorials/ticket-app/frontend/utils/createTickDrop.js#L64-L73
+```
+
+Once both the series and drop are defined, `claim` can be called and will mint the desired NFT POAP. 
+
+---
+
+## Creating Linkdrops
```

Comment:
This section needs to be revised - it looks like a copy paste from other tutorials